### PR TITLE
Add prod context.

### PR DESCRIPTION
**Pull request by afwillia**
PR Checklist:
Update dev to lastest DCA image, v0.2.13-beta. Add prod context to `cdk.json`

### CODE REVIEW

File: cdk.json [hunk 1]
```diff
@@ -31,13 +31,22 @@
...
       "VPC_CIDR": "10.255.70.0/24"
+    },
+    "prod": {
+      "IMAGE_PATH_AND_TAG": "ghcr.io/sage-bionetworks/data_curator:0.2.13-beta",
+      "AWS_DEFAULT_REGION": "us-east-1",
+      "PORT": "3838",
+      "COST_CENTER": "NO PROGRAM / 000000",
+      "STACK_NAME_PREFIX": "dca",
+      "ACM_CERT_ARN": "arn:aws:acm:us-east-1:878654265857:certificate/a3b7c804-c1fc-4e58-bbc6-541ef2d65816",
+      "VPC_CIDR": "10.255.70.0/24"
```
Comment (ConsoleCatzirl) [inline cdk.json]: should this be the same IP range as dev?

Comment (afwillia) [reply cdk.json]: @zaro0508 what do you think? We only mentioned the ARN when we met earlier.

Comment (zaro0508) [reply cdk.json]: every VPC needs to have a unique CIDR.  Let's go with `10.255.71.0/24`

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: cdk.json (hunk 1 above)
Comment (afwillia) [reply cdk.json]: thanks, just updated that!

**Review by ConsoleCatzirl — APPROVED** [review]

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**